includes/ReverseWithdrawal/ReverseWithdrawal.php

```
@@ -35,8 +35,10 @@ public function __clone() {
      * Unserializing instances of this class is forbidden.
      *
      * @since 3.5.1
+     *
+     * @param array $data Unserialized data.
      */
-    public function __wakeup() {
+    public function __unserialize( array $data ): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$data'. (undefined)

(UnusedFormalParameter)